<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting_window</class>
 <widget class="QMainWindow" name="setting_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="menu_btn_1">
         <property name="font">
          <font>
           <family>ONE Mobile POP</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>업데이트</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>481</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_btn">
         <property name="font">
          <font>
           <family>ONE Mobile POP</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>저장</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit_btn">
         <property name="font">
          <font>
           <family>ONE Mobile POP</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>닫기</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="setting_stacks">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>ONE Mobile POP</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <widget class="QWidget" name="page_update">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>현재 버전</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_version">
            <property name="font">
             <font>
              <family>ONE Mobile POP</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>임시 텍스트</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>24시간마다 최신 버전 체크</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckButton" name="setting_update_program">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>ONE Mobile POP</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>24시간마다 드랍 테이블 업데이트</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>ONE Mobile POP</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>(10초 정도 소요됩니다)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckButton" name="setting_update_drop">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>ONE Mobile POP</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>50</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>수동 업데이트</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="update_btn_program">
               <property name="font">
                <font>
                 <family>ONE Mobile POP</family>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="text">
                <string>프로그램 버전</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="update_btn_master">
               <property name="font">
                <font>
                 <family>ONE Mobile POP</family>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="text">
                <string>master.db</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="update_btn_drop">
               <property name="font">
                <font>
                 <family>ONE Mobile POP</family>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="text">
                <string>드랍 테이블</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>ONE Mobile POP</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>※ master.db : 신규 사도 / 랭크 생길 때 업데이트하는 파일</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>263</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCheckButton</class>
   <extends>QPushButton</extends>
   <header>widgets/wrapper/misc.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
